backends/ebpf/runtime/ebpf_kernel.h

```diff
@@ -1,17 +1,14 @@
 /*
 Copyright 2018 VMware, Inc.
+SPDX-License-Identifier: (GPL-2.0-only or Apache-2.0)
```

Comment:
Technically we also use libbpf to build the binaries for the kernel, which also has a complicated but more permissive license: 

https://github.com/libbpf/libbpf?tab=readme-ov-file#license

I do not know whether that has any implications, quite the headache.

Reply:
The license there appears to be the user's choice of BSD-2-Clause or LGPL-2.1.  That is compatible with using it from a GPL-2.0-only program, or from an Apache-2.0 program, which are all we ever care about (and rarely if ever the GPL-2.0-only choice, but nice to know it is covered.)

The good news for the headache is that we can document it once, and the set of files involved should change extremely rarely, so we can automate simple license checks that these files continue to have the licenses we think are safe and defensible, and then the headache is past :-)